opal/mca/common/ofi/common_ofi.h

```diff
@@ -26,8 +26,6 @@ BEGIN_C_DECLS
 typedef struct opal_common_ofi_module {
     char **prov_include;
     char **prov_exclude;
-    int verbose;
```

Comment:
This change isn't mentioned in the commit message, probably belongs in one of the cleanup commits